### ISSUE TITLE
Accessibility: allow configuring mechs and truepath ships without hovering

### ISSUE THREAD

**Issue by Keithcat1** [pmotschmann/Evolve#1065]
Currently, configuring mechs or truepath ships to be built seems to require hovering over the menus to open them, simply pressing enter doesn't work. This is a problem because screen readers grant poor control over the mouse at the best of times, allowing it to disappear at any moment and take the menu with it. Not every attempt to put the mouse on a menu will have any effect. Ideally, these menus would work like the menus in the settings tab, just requiring me to press enter and not randomly closing. There ccould be an option for it in settings.
